### DISCUSSION TITLE
TrackHat Input Custom Not Working

### DISCUSSION THREAD

**Discussion by pugnaproveritas** [opentrack/opentrack#1474] [General]
Selecting the Tracker model as "custom" under the TrackHat input does not allow you to input your custom LED tracker dimensions. It seems that Custom continues with the settings of what was selected before. 

E.g. if I select mini clip (left) then, select custom, the movement tracked is no different from when mini clip was selected. The same applies if I select clip (left). 

This does not happen when I switch between clip (left) and mini clip (left) which leads me to think there is an issue with the "custom" setting as currently implemented.

![image](https://user-images.githubusercontent.com/105658320/168623871-13da450c-5f1a-43d5-bb9a-9b965c3c48f1.png)

**Comment by sthalik** [comment]
For a custom model, you need to take your PT settings:

```ini
clip-bz=40
clip-tz=20
active-model-panel=0
```

then add them to the `tracker-trackhat` group after prepending `pt\` to each line.

```ini
pt\clip-bz=40
pt\clip-tz=20
pt\active-model-panel=0
```

But there are some bugs at the moment:

- head center can't be set this way, it's always forced to left clip
- cap or custom model can't be selected as custom due to `active_model_panel=0` being forced by accident

If you can get the basic functionality to work then I should provide a temporary build until a fix finds its way to a release build.

**Reply by pugnaproveritas** [reply]
Sorry but I'm not familiar with how to do this, would it be possible further down the line to include the UI we see in the PointTracker options like below? As things currently stands the basic functionality is there but with the wrong tracker dimensions, the tracking output is quite jerky even with accela smoothing maxed out.

![image](https://user-images.githubusercontent.com/105658320/168742907-902e4b64-0ed9-416a-b5c6-c02d7ffaff16.png)
